src/server.js

```diff
@@ -0,0 +1,40 @@
+const FeedParser = require('feedparser')
+const request = require('request')
+const Rx = require('rx') 
+
+// Event loop will run 3 times for testing
+const interval = Rx.Observable.interval(5000).timeInterval().take(3)
+
+const eventLoop = interval.subscribe(
+    function(val) {
+        // Log what loop interval we are on
+        console.log(val)
+
+        let req = request('https://pypi.python.org/pypi?%3Aaction=rss')
```

Comment:
Should prefer `const` over `let` where possible. Might be worth adding an eslint pref-const rule (http://eslint.org/docs/rules/prefer-const).